app/serializers/activitypub/note_serializer.rb

```diff
@@ -142,21 +142,6 @@ def name
     end
   end
 
-  class CustomEmojiSerializer < ActiveModel::Serializer
-    include RoutingHelper
-
-    attributes :type, :href, :name
-
-    def type
-      'Emoji'
-    end
-
-    def href
-      full_asset_url(object.image.url)
-    end
-
-    def name
-      ":#{object.shortcode}:"
-    end
+  class CustomEmojiSerializer < ActivityPub::EmojiSerializer
```

Comment:
what's the difference between this and the Emoji serializer? do we need both?

Reply:
AMS derives serializer name from class name, virtual_tags is array of various objects, some of which are instances of CustomEmoji, but what we need is called ActivityPub::EmojiSerializer. It's easier to do this than define an AMS method that changes how serializer names are derived.

Reply:
Does the `def object_type; emoji; end` thing mean this isn't relevant? having a hard time finding the AMS docs on this, sorry.....